### PR TITLE
About Me Project Parts 1-3

### PR DESCRIPTION
**Pull request by eyobtefera**
This is the about me project parts 1-3.

### CODE REVIEW

**Comment by eyobtefera** [comment]
Part 4 of the about me section

File: .classpath [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (nsallembiencodeu) [inline .classpath:1]: It's probably OK to remove the .classpath, .settings/..., .project, etc. files which are Eclipse specific from the repo. The idea here is that everyone should be able to use whatever editor they want (Eclipse, IntelliJ, VIM, Emacs, ...) and that the files that contain configuration for the editor should be be in the repo, but stay on your local working directory.

**Review by nsallembiencodeu — CHANGES_REQUESTED** [review]
The changes to the main code look fine, however there are a lot of superfluous files which make it difficult to review the change. Would you be able to remove them from the PR?

File: .classpath (hunk 1 above)
Comment (eyobtefera) [reply .classpath:1]: Okay, my problem was without those changes nothing would work locally so I figured that they were necessary to commit. From now on I will not including those changes. Thank you for your feedback.

**Comment by nsallembiencodeu** [comment]
Submitted the other PR which had the same changes

**Pull request closed without merging**